vars/loadResources.groovy

```diff
@@ -0,0 +1,16 @@
+def call(Map args = [:]) {
+    if(!args.file) {
+        error "Missing manadatory parameter: file"
+    }
+
+    def resources = readYaml file: args.file
+
+    if (resources instanceof LinkedList) {
```

Comment:
Can you not [resources].flatten().groupBy({ it.kind })

Reply:
getting exception `No signature of method: java.util.LinkedHashMap.flatten()`

Reply:
can do for other two cases